modules/ROOT/pages/asm-secret-group-creation-task.adoc

```diff
@@ -1,112 +1,161 @@
+. Select *Certificate Pinset* in the menu on the left, and click *Add Certificate Pinset*. 
+. In the *Certificate Pinset* screen, add the required information. +
+* *Name* - Enter a name for the certificate pinset. 
+* *Certificate File* - Click *Choose File* and select the certificate file to upload.
+* *Expiration Date* (Optional) - Select the expiration date for the certificate.
+// _COMBAK: What type of certificate to upload?
 
-== Adding a Shared Secret
+== Add a Shared Secret
 
-. In the secret group to which you want to add this secret, select Shared Secret, and click the Add Shared Secret button.
-. In the Name field, write a name for your shared secret.
-. In the Type field, select the shared secret type from the drop-down menu.
+. In the *Secret Groups* list view, select the secret group to which to add a shared secret. 
+. Select *Shared Secret* in the menu on the left, and click *Add Shared Secret*. 
+. In the *Add Shared Secret* screen, add the required information. 
 +
-* Username Password: +
-If you select this type, you need to provide a username and password
-* Symetric Key: +
-If you select this type, you need to provide a Base64 string containing symmetric key.
-* S3 Credential: +
-If you select this type, you need to provide the access key ID and the secret access key to an S3 bucket.
-* Blob: +
-If you sleect this type, you need to provide a base64-encoded value.
+* *Name* - Enter a name for your shared secret.
+* *Type* - Select the shared secret type from the drop-down menu.
+ ** *Username Password* - Provide a username and password.
+ ** *Symmetric Key* - Provide a Base64 string containing symmetric key.
+ ** *S3 Credential* - Provide the access key ID and the secret access key to an S3 bucket.
+ ** *Blob* - Provide a base64-encoded value.
+. Click *Save*. 
 
-== Adding a Certificate Revocation List Distribution Configuration
+== Add a Certificate Revocation List Distributor Configuration
 
-. In the secret group to which you want to add this secret, select CRL Distribution Point, and click the Add CRL button.
-. In the name field, write a name for your CRL distribution point.
-. In the Distributor Certificate field, select your CRL distributor from the drop-down menu. +
-//_COMBAK: How to populate this drop-down.
-. In the CA Certificate field, select the CA certificate to query against the CRL distributor from the drop-down menu. +
-//_COMBAK: This can be a URL, FTP, and LDAP?
-. In the Frequency field, determine the interval in minutes to query the CRL distributor.
-. In the Complete CRL Issuer URL, add the CRL issuer URL. +
-Additionally, you can add a Delta CRL in the Delta CRL Issuer URL field.
+Add a list of certificates that were revoked by certificate authorities before their expiration dates.
 
-== Adding a TLS Context
+. In the *Secret Groups* list view, select the secret group to which to add a Certificate List Distributor (CRL) configuration. 
```

Comment:
Do we explain anywhere what "Distributor configuration" means?

Reply:
It's Step 3